app/models/offer.js

```diff
@@ -173,187 +181,194 @@ export default DS.Model.extend({
-    return this.get('crossroadsTransport') && this.get('crossroadsTransport.isVanAllowed')
+  hasCrossroadsTransport: Ember.computed('crossroadsTransport', function() {
+    return this.get('crossroadsTransport') && this.get('crossroadsTransport.isVanAllowed');
   }),
 
-  hasGogovanTransport: Ember.computed('gogovanTransport', function(){
+  hasGogovanTransport: Ember.computed('gogovanTransport', function() {
     return this.get('gogovanTransport') && !this.get('gogovanTransport.disabled');
   }),
 
   // display "General Messages Thread"
-  displayGeneralMessages: Ember.computed('isDraft', 'lastMessage', function(){
+  displayGeneralMessages: Ember.computed('isDraft', 'lastMessage', function() {
     return !(this.get('isDraft') && this.get('lastMessage') === null);
   }),
 
   // to sort on offer-details page for updated-offer and latest-message
-  latestUpdatedTime: Ember.computed('lastMessage', function(){
+  latestUpdatedTime: Ember.computed('lastMessage', function() {
     var value;
-    switch(Ember.compare(this.get('lastMessage.createdAt'), this.get('updatedAt'))) {
-      case 0 :
-      case 1 : value = this.get('lastMessage.createdAt'); break;
-      case -1 : value = this.get('updatedAt'); break;
+    switch (Ember.compare(this.get('lastMessage.createdAt'), this.get('updatedAt'))) {
+      case 0:
+      case 1:
+        value = this.get('lastMessage.createdAt');
+        break;
+      case -1:
+        value = this.get('updatedAt');
+        break;
     }
     return value;
   }),
 
-  hasCompleteGGVOrder: Ember.computed('delivery.gogovanOrder.status', function(){
+  hasCompleteGGVOrder: Ember.computed('delivery.gogovanOrder.status', function() {
     return (this.get("delivery.gogovanOrder.status") || "") === "completed";
   }),
 
-  showOfferIcons: Ember.computed('hasCompleteGGVOrder','itemCount', 'isClosed', 'hasReceived', function(){
+  showOfferIcons: Ember.computed('hasCompleteGGVOrder', 'itemCount', 'isClosed', 'hasReceived', function() {
     return this.get("itemCount") > 0 && !(this.get('isClosed') || this.get('hasReceived')) && !this.get("hasCompleteGGVOrder");
   }),
 
-  statusBarClass: Ember.computed('state', function(){
-    if(this.get("isSubmitted")){ return "is-submitted"}
-    else if(this.get("isUnderReview")){ return "is-under-review"}
-    else if(this.get("isReviewed")){return "is-reviewed"}
-    else if(this.get("isScheduled")){return "is-scheduled"}
-    else if(this.get("isClosed")){return "is-closed"}
-    else if(this.get("hasReceived")){return "is-received"}
+  statusBarClass: Ember.computed('state', function() {
+    if (this.get("isSubmitted")) { return "is-submitted"; } else if (this.get("isUnderReview")) { return "is-under-review"; } else if (this.get("isReviewed")) { return "is-reviewed"; } else if (this.get("isScheduled")) { return "is-scheduled"; } else if (this.get("isClosed")) { return "is-closed"; } else if (this.get("hasReceived")) { return "is-received"; }
```

Comment:
I feel it looks difficult for readability purpose. 

Can we do like this ?

```
if (foo === 1) {
  bar = 2;
} else if (foo === 2) {
  bar = 1;
} else {
  bar = 3;
}
```
https://github.com/emberjs/ember.js/blob/master/STYLEGUIDE.md#block-statements

Just a suggestion. Correct me if I am wrong.

Thanks.

Reply:
Yes agree with you actually it is my sublime settings which has made it in one like. Actual fix I did to add Semi-colon ";" 😄